groups/stack/module-ecs-stack/load-balancer.tf

```diff
@@ -21,6 +21,22 @@ resource "aws_lb_listener" "admin-web-lb-listener" {
   }
 }
 
+resource "aws_lb_listener" "admin-web-lb-listener" {
```

Comment:
the resource name "admin-web-lb-listener" has already been used by the https listener above so clashes when running the terraform

changing this line to

```resource "aws_lb_listener" "admin-web-lb-http-listener" {```

should let it run through without error